src/status_im/contexts/wallet/common/utils.cljs

```diff
@@ -177,56 +168,6 @@
       (str $ address))
     address))
 
-(def id->network
```

Comment:
I had my gripes about a file called common/utils.cljs as it has and will grow to become a random bucket of functionality. 
I moved these to `status-im.contexts.wallet.common.utils.networks` as it is much better to categorise utilities.
I would move more but I wanted to reduce the noise, would be nice if someone else could move some of these pieces in their prs too 👍